config/config.msft.yaml

```diff
@@ -132,7 +132,7 @@ clouds:
       maestro:
         imageTag: bc2f131579c6ffc664c15f48c50a9936f1b4a7ce
       clusterService:
-        imageTag: ecd15ad
+        imageTag: 9f7fef3
```

Comment:
The one that should have been updated is config/config.yaml and not config/config.msft.yaml. We can update this one anyway too as I think it's not used yet.

Reply:
fixed!